osa/provenance/utils.py

```diff
@@ -124,7 +124,7 @@ def parse_variables(class_instance):
 
         class_instance.Analysisconfigfile_dl1 = os.path.realpath(configfile_dl1)
         class_instance.ObservationRun = class_instance.args[0].split(".")[0]
-        class_instance.PedestalCleaning = "True"
+        class_instance.PedestalCleaning = "False"
```

Comment:
Why False instead? Pedestal cleaning will be True by default

Reply:
ah sorry !
I though it was False by default.. then I see two options:
- completely remove PedestalCleaning from the provenance 
- close this PR.

Reply:
Is there a track of this config option in lstchain? If so I would say we could remove it from the osa provenance. In principle, lstosa will not set up this option from now on.

Reply:
actually there is no argument `pedestal-cleaning` aymore in `lstchain_dl1ab.py`
ok, I'm going to remove this from the provenance..

Reply:
done.

Reply:
There is no pedestal-cleaning argument anymore, but there is `--no-pedestal-cleaning` to disable it. I guess this info is kept somewhere in the metadata info of the files along with the config parameters used?

Reply:
It does not seem to be the case, all the content of the config file is kept, which means all the values for the thresholds :(

Anyhow, I think this is something that has to be done in `lstchain_dl1ab.py`, lstosa does not set up this option.